tests/qos/test_qos_sai.py

```diff
@@ -63,8 +63,10 @@ def testParameter(
         logger.info("qosConfig {}".format(dutQosConfig))
 
     @pytest.mark.parametrize("xoffProfile", ["xoff_1", "xoff_2"])
-    def testQosSaiPfcXoffLimit(self, xoffProfile, ptfhost, dutTestParams, dutConfig, dutQosConfig,
-                               ingressLosslessProfile, egressLosslessProfile):
+    def testQosSaiPfcXoffLimit(
+        self, xoffProfile, ptfhost, dutTestParams, dutConfig, dutQosConfig,
+        ingressLosslessProfile, egressLosslessProfile, ssh_tunnel_to_syncd_rpc
```

Comment:
Why is ssh_tunnel fixture used for each testcase? Do you need to setup/teardown for each test? If so, then it has to be a func level fixture. 
when running pytest, you can use --setup-show option that will show the setup/teardown for each of the fixtures

Reply:
It's not required to be run for every test case. Looks like 'arpPopulate' fixture is set to 'autouse', which will ensure 'ssh_tunnel' fixture will run, so this can be removed for test cases. Will remove it.